elastic/apm-server
# smoketest: Improve output, parallelize by test

## Motivation/summary

Improves the smoke test output by redirecting all terraform commands to
a `tf.log` file, and changes the way the tests are discovered to allow
each test to run in parallel.

## Checklist

~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)~
~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~
~- [ ] Documentation has been updated~

## How to test these changes

1. `make smoketest/all SMOKETEST_VERSION=7.17,latest`
2. Verify the output is easily readable.

## Related issues

#8303
